### PR TITLE
update kubelogin-poc

### PR DESCRIPTION
**Pull request by tuminoid**
Update kubelogic-poc. Remove other connectors than LDAP/static and document it and its configuration better.

Add `run.sh` to replicate the experiment easily.

### CODE REVIEW

File: security/kubelogin-poc/kind.conf [hunk 1]
```diff
@@ -19,9 +19,10 @@ nodes:
     kind: ClusterConfiguration
     apiServer:
         extraArgs:
+            v: "1"
             oidc-issuer-url: https://dex.example.com:32000
             oidc-client-id: kubelogin-test
-            oidc-ca-file: /etc/ssl/certs/dex-test/dex-ca-test.pem
+            oidc-ca-file: /etc/ssl/certs/dex-test/ca.pem
             oidc-username-claim: email
             # group-claim: cn is test related as well, normally it would say "groups"
```
Comment (lentzi90) [inline security/kubelogin-poc/kind.conf:27]: nit: this comment is no longer relevant

File: security/kubelogin-poc/run.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+kubectl apply -f k8s/dex.yaml
+kubectl create secret tls dex.example.com.tls \
+    --cert=ssl/cert.pem --key=ssl/key.pem --namespace=dex
+kubectl apply -f k8s/openldap.yaml
+
+# create role for user "customuser"
+# user is whatever the group-claim=groups field defines groups field is
+kubectl create clusterrolebinding oidc-cluster-admin \
+    --clusterrole=cluster-admin \
+    --user='Bar1'
```
Comment (lentzi90) [inline security/kubelogin-poc/run.sh:25]: Typo?
```suggestion
    --user='customuser'
```

File: security/kubelogin-poc/run.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+# create role for user "foo"
+kubectl create clusterrolebinding oidc-view-group \
+    --clusterrole=view \
+    --user='Bar2'
```
Comment (lentzi90) [inline security/kubelogin-poc/run.sh:30]:
```suggestion
    --user='foo'
```

File: security/kubelogin-poc/kind.conf (hunk 1 above)
Comment (tuminoid) [reply security/kubelogin-poc/kind.conf:27]: Fixed.

File: security/kubelogin-poc/run.sh (hunk 2 above)
Comment (lentzi90) [reply security/kubelogin-poc/run.sh:25]: Ah I see! Thanks for explaining

**Review by lentzi90 — APPROVED** [review]
/approve

**Review by Rozzii — COMMENTED** [review]
/lgtm

**Pull request merged**